src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java

```diff
@@ -113,4 +116,15 @@ public void shouldProcessDependantResponseFromJson() throws Exception {
         assertThat(output).isEqualTo(expected);
     }
 
+    @Test(expected = InvalidApplicationException.class)
+    public void shouldThrowExceptionOnInvalidJson() {
+        processor.fromJson("{asads:", ResponseData.class);
+    }
+
+    @Test(expected = InvalidApplicationException.class)
+    public void shouldThrowExceptionOnInvalidJsonWithTypeReference() {
```

Comment:
Does this test add anything beside the first one?

Reply:
Calls a different overloaded method